modules/ocf/templates/kubectl/kubectl.conf.erb

```diff
@@ -0,0 +1,20 @@
+users:
+- name: ocfstaffer
+  user:
+    exec:
+      command: "/opt/share/utils/sbin/gen-kubernetes-cert"
+      apiVersion: "client.authentication.k8s.io/v1beta1"
+
+clusters:
+- name: prod
+  cluster:
+    certificate-authority-data: <%= @cluster_cert_base64.gsub(/\n/, "") %>
+    server: https://kubernetes:6443
+
+contexts:
+- name: prod
+  context:
+    cluster: prod
+    user: ocfstaffer
+
+current-context: prod
```

Comment:
Not sure how this will work if staffers want to add other contexts in their local kubeconfig file and switch to them. We might be better off just telling people "run `kubectl config use-context prod` to access prod".

Reply:
I'd rather not have to opt-in to use default prod settings. Why can't staffers point to their own customized kubeconfig

Reply:
What other contexts would we want to use? I couldn't think of any, so I figured a system-wide config was OK.

That being said, if this comes up in the future (for example, if we want a separate context for cluster admin privileges), we can [merge configuration files](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) between a system-wide and user-specific config. In that case, the user config would override the `current-context` given here.